### PR TITLE
add support for .NET 7

### PR DESCRIPTION
**Pull request by Ne4to**

### CODE REVIEW

**Comment by Ne4to** [comment]
Only `Failing for demo purposes` tests fail. I'm not sure if I should somehow modify CI to exclude them or not.

**Comment by gfoidl** [comment]
I didn't understand why CI failed, so made some updates with https://github.com/gfoidl/trx2junit/pull/114. The demo-failures should be excluded in the setup.

Can you please sync with latest main / master branch, then CI should pass and we can go ahead and merge.

Besides that: is there a reason why a explicit .NET 7 target is needed? Especially as the project has https://github.com/gfoidl/trx2junit/blob/666e1c13991953e1e625ba5b581c7fb4784f6504/source/trx2junit/trx2junit.csproj#L7 so it should just work on .NET 7 w/o any change -- but I don't know about pre-releases.
(I'm just asking to learn if there's some cornercases, etc. -- I won't push back the change per se).

Anyway: thanks for the contribution!

**Comment by gfoidl** [comment]
CI is happy now.

Please answer the question, then I'll merge.

**Comment by gfoidl** [comment]
Another request, please add an entry to https://github.com/gfoidl/trx2junit/blob/666e1c13991953e1e625ba5b581c7fb4784f6504/ReadMe.md?plain=1#L85-L88

**Comment by Ne4to** [comment]
It looks like it is not required actually. 😁 
I was migrating a project to .NET 7 and it fails on CI.
I didn't investigated it a lot just noticed that log ended with some message about .NET 6 and trx2junit.
I remember that last time I migrated project to .NET 6 it required a new version of trx2junit.
So now I just blamed trx2junit for no reason, sorry. 😔

There are two reasons why my CI failed.
1 - It was using .NET 6 instead of .NET 7
2 - `dotnet test` parameters were changed.

My CI script:
```
dotnet test ./MyProject.sln -c Release -l trx -r ./results --filter Category!=Integration
trx2junit ./results/*.trx --output ./results/junit/
```

Notice `-r ./results` parameter.

 
`dotnet test --help` output for .NET 6
> -r, --results-directory <RESULTS_DIR>                      The directory where the test results will be placed.
>                                                            The specified directory will be created if it does not exist.
														   

`dotnet test --help` output for .NET 7
> -r, --runtime <RUNTIME_IDENTIFIER>                         The target runtime to test for.

~~I'm still investigating how to modify my CI script. I removed `-r ./results` and now it saves `*.trx` to several test projects subfolders `TestResults`.~~

**UPD:**
now it is `--results-directory ./results`

File: ReadMe.md [hunk 1]
```diff
@@ -83,13 +83,14 @@ echo 'export PATH="$PATH:/root/.dotnet/tools"' >> "$BASH_ENV"
...
 In order to install this tool a [.NET SDK](https://dotnet.microsoft.com/download/dotnet) must be present. Supported SDKs are:
+* .NET 7.0
 * .NET 6.0
 * .NET 5.0
 * .NET Core 3.1
 
 # Core functionality as standalone package trx2junit.Core
 
-Starting with v2.0.0 it's possible to use the core functionality as standalone package `trx2junit.Core`.  
+Starting with v2.0.0 it's possible to use the core functionality as standalone package `trx2junit.Core`.
```
Comment (gfoidl) [inline ReadMe.md]:
```suggestion
Starting with v2.0.0 it's possible to use the core functionality as standalone package `trx2junit.Core`.  
```

**Comment by gfoidl** [comment]
😄 I spotted https://github.com/dotnet/docs/pull/32163 a few minutes ago (which is what bit you).

Even if not needed I'm going to merge this PR, as it opens the door for https://github.com/gfoidl/trx2junit/issues/115.

Thank you!

**Pull request merged**